docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,35 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename tag
+
+*MSS*
+
+. User requests to rename tag.
+. AddressBook shows a list of unique tags.
+. User requests to rename a specific tag in the list.
+. AddressBook prompts for new tag to rename selected tag to from User.
+. User enters new tag.
+. AddressBook prompts for confirmation from User.
```

Comment:
If you have this step, the following should be `User confirms the rename operation` or something on similar lines.
Remember, use case depicts the interaction between the user and the system.